### ISSUE TITLE
Report an issue: urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:24.70-24.100

### ISSUE THREAD

**Issue by leonsaeon** [PerseusDL/morpheus#25]
[tlg0012.tlg001.perseus-grc2.xml](https://github.com/PerseusDL/canonical-greekLit/tree/58d5346/data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml)

In Il. 24.79, ἔνθορε is incorrectly parsed with this: 

ἔνθορος adjective 1st & 2nd
ἐνθορ-ε
vocative singular masculine
ἐνθορ-ε
vocative singular feminine 

ἔνθορος isn't a word. The parsing should give the aorist of ἐνθρῴσκω instead.

**Comment by leonsaeon** [comment]
In Il. 24.83, σπῆι is incorrectly glossed as coming from ἕπομαι or σπάω, when it's a form a σπέος.

**Comment by leonsaeon** [comment]
In IL. 24.84, εἵαθ' (εἵατο), from ἧμαι, has no info at all.

**Comment by lcerrato** [comment]
@leonsaeon 
Thank you for these reports.
Parsing and word analysis is unrelated to this repository. I will move the issues to the morpheus reports.
https://github.com/PerseusDL/morpheus/issues/20

**Comment by lcerrato** [comment]
Noted in larger issue #20

**Comment by helmadik** [comment]
> [tlg0012.tlg001.perseus-grc2.xml](https://github.com/PerseusDL/canonical-greekLit/tree/58d5346/data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml)
> 
> In Il. 24.79, ἔνθορε is incorrectly parsed with this:
> 
> ἔνθορος adjective 1st & 2nd ἐνθορ-ε vocative singular masculine ἐνθορ-ε vocative singular feminine
> 
> ἔνθορος isn't a word. The parsing should give the aorist of ἐνθρῴσκω instead.

As long as ἔνθορος is listed in dictionaries, even as a varia lectio, it should also be in morpheus. So don't change this, I'd say. But do make sure ἐνθρῴσκω is there as well.

**Comment by lcerrato** [comment]
Updated in main issue (https://github.com/PerseusDL/morpheus/issues/20)

**Comment by leonsaeon** [comment]
I'm sorry, I don't know why I would have said that it's not a word, rather
than not a *Homeric* word. My more important point is that the form in
Homer isn't from ἔνθορος, but from ἐνθρῴσκω (and this is not disputed). I
hope that bit is useful!

**Comment by helmadik** [comment]
yes, helpful! In case someone was working through these, I just wanted to add that ἔνθορος should be given the benefit of the doubt by Perseus (as long as it's listed by lexica). I'm trying to clean up morpheus and now that I've taken care of my own pet peeves, I'm glad to see the issues reported here. Many like ἔνθορε I've already taken care of locally but there's still more to do. Pull requests to follow. Please check in at https://anastrophe.uchicago.edu/morpheus?word=ἔνθορε&input=1 in case you have more. Happy to get more reports there, but I'll follow the morpheus issues thread as well.
